index.html

```diff
@@ -26,22 +28,151 @@
+      </div>
+
+        <div>
+        <img class = "placeholder" src="PImages/Img Placeholder.png" alt=""></div>
+            <div id="multip" class="multipost">
+                <h2 id="multi" class="multi">Multi-Post Stories</h2>
+                <p id="stories" class="stories">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            </div>
+            <div id="buttons1" class="buttonss1">
+                <button type="button" class="btn btn-primary btn-lg">CSS</button>
+                <button type="button" class="btn btn-primary btn-lg">HTML</button>
+                <button type="button" class="btn btn-primary btn-lg">BootStrap</button>
+                <button type="button" class="btn btn-primary btn-lg">Ruby</button>
+            </div>
+            <buttons id="seeproject" class="btn2 btn-primary btn-lg">See Project</buttons>
+    </section>
+    <section class="cardworks" id="cards">
+      <ul class = cardsarr>  
+        <li class = "cardlist"><div id="cwaction1" class="cwaction1">
+            <h2 class="card1">Professional Art Printing Data</h2>
+            <p class="cardtext">Profesional Art Printing DataA daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <button type="button" class="btn3 btn-primary btn-lg">HTML</button>
+              <button type="button" class="btn3 btn-primary btn-lg">BootStrap</button>
+                <button type="button" class="btn3 btn-primary btn-lg">Ruby</button>
+                <div class ="footer"><button class = "projects btn-primary btn-lg">See project</button></div>
+                </div>
+            </li>
+                <li class="cardlist"><div id="cwaction1" class="cwaction1">
+                    <h2 class="card1">Professional Art Printing Data</h2>
+                    <p class="cardtext">Profesional Art Printing DataA daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                    <button type="button" class="btn3 btn-primary btn-lg">HTML</button>
+                        <button type="button" class="btn3 btn-primary btn-lg">BootStrap</button>
+                        <button type="button" class="btn3 btn-primary btn-lg">Ruby</button>
+                       <div class ="footer"><button class = "projects btn-primary btn-lg">See project</button></div>
+                    </div></li>
+
+                        <li class="cardlist"> <div id="cwaction1" class="cwaction1">
+                            <h2 class="card1">Professional Art Printing Data</h2>
+                            <p class="cardtext">Profesional Art Printing DataA daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
```

Comment:
- [ ] All the cards except the first one are not showing up in your design. kindly consider fixing them. right now the elements in these cards are not showing up.
<img width="252" alt="Screenshot 2023-06-04 at 3 25 01 AM" src="https://github.com/Zuwairanajma/Portfolio-Projects/assets/96683843/97d62b7a-372a-4e65-aa69-1943450ce390">